src/processes.c

```diff
@@ -2029,6 +2029,8 @@ static int ps_read (void)
 	int stopped  = 0;
 	int paging   = 0;
 	int blocked  = 0;
+        int num_cmdline = 0;
+        int count = 0;
```

Comment:
If count is never used directly with the exception of subtracting, I feel that it's more clear to just maintain two counters, or make this more specific such as "num_cmd".

Reply:
So in general I definitely agree with you. But I chose to go for consistency in this case with the other blocks.